src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// <copyright file="OtlpLogExporterHelperExtensions.cs" company="OpenTelemetry Authors">
```

Comment:
Remove `InternalsVisibleTo` entry for `OpenTelemetry.Exporter.OpenTelemetry.Logs`.